FMUv3_REV_D/README.md

```diff
@@ -0,0 +1,22 @@
+# FMUv3 Pixhawk Open Hardware Design
+
+FMUv3 was introduced in 2015. Identical to FMUv2, it doubled the usable flash to 2MB.
+
+The FMUv3 design is used in the Pixhawk 2 FC board.
+This hardware is the result of a collaboration of the open hardware development community and 3D Robotics.
+
+> **Tip** General information about designs, schematics, reference hardware, FMU versions, licensing, and trademarks can be found in the repo [README](../README.md).
+
+
+## Design Files
+
+The design is provided in [Altium Designer](https://www.altium.com/) CAD files.
+
+It consists of schematic files (*.SchDoc), BOM listings, and includes open reference hardware PCB layouts (.PrjPcb). 
+
+> **Tip** PDF versions of the schematics and PCB layouts has been made available for easy previewing (i.e. if you don't have Altium Designer).
+
+
+<!-- Removed text: FMUv3 open design (along with IMUv3 + PSMv3 support components) -->
```

Comment:
I removed this text from the home page description. I presume this is hardware in the folders with similar names. Can you explain briefly what they are for?